api/signup.go

```diff
@@ -163,10 +163,14 @@ func (a *Api) sendSignUpInformation(res http.ResponseWriter, req *http.Request,
 // @Description  This post is sent by the signup logic. In this state, the user account has been created but has a flag that
 // @Description  forces the user to the confirmation-required page until the signup has been confirmed.
 // @Description  It sends an email that contains a random confirmation link
+// @Description  The email is sent in the language defined by "x-tidepool-language" header
+// @Description  Otherwise if null "Accept-Language" header otherwise if null "ENglish"
```

Comment:
```suggestion
// @Description  Otherwise if null "Accept-Language" header otherwise if null "English"
```
?

Reply:
this one is on purpose as we actually return "EN" for "English"